test/pull-variables-used-twice/expected-stderr

```diff
@@ -0,0 +1 @@
+Warning: config "branch:%s/component:keboola.variables/config:%s" have 2 "variablesFor" relations [config "branch:%s/component:ex-generic-v2/config:%s"; config "branch:%s/component:ex-generic-v2/config:%s"], but only one is expected
```

Comment:
Funkcionalny test pre dalsi edge case:
- moze sa stat (API to nevaliduje), ze 2 konfiguracie obsahuju rovnake `variablesId`
- moze to nastat napr. pri klonovani konfiguracie v UI (nie je to atomicke)
- ak sa to stane, tak nevieme pod ktoru konfiguraciu `variables` ulozit
- v tom pripade vypiseme `warning`, a `variables` ulozime ako klasicku konfiguraciu, do samostatneho adresara -> tj. `Relation` sa ignoruje